php/index.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+require_once("configuration.php");
+
+$base_url = $hostname . $version . $api;
+$access_token_url = $base_url . $token_endpoint;
+
+$access_token_response = getAccessToken($access_token_url, $client_id, $client_secret);
+
+// You will want to cache this access token (it's good for 24 hrs)
+$access_token = $access_token_response->access_token;
+
+$locations_url = $base_url . $locations_endpoint;
+$params = array("q" => "library");
+
+$resp = getLocations($locations_url, $access_token, $params);
```

Comment:
Verbose variable names seems to be the convention for this sample code, is there a reason $resp was used instead of  $response or $locations_response?